docs/env.md

```diff
@@ -1,117 +1,260 @@
+- Default: `"marsha"`
 
-:   Boolean
+#### POSTGRES_HOST
 
-Mandatory
+Address for the Postgres database used by Marsha.
 
-:   No
+- Type: string
+- Required: No
+- Default: `"localhost"`
 
-Default
+#### POSTGRES_PASSWORD
 
-:   `True` if `DJANGO_CONFIGURATION` is set to `Development`, `False`
-    otherwise
+Password corresponding to the user specified in `POSTGRES_USER`, for the Postgres database used by Marsha.
 
-Choices
+- Type: string
+- Required: No
+- Default: `"pass"`
 
-:   `True` or `False`
+#### POSTGRES_PORT
 
+Port for the Postgres database used by Marsha.
 
-## DATABASE_URL
+- Type: number
+- Required: No
+- Default: `5432"`
 
-Description
+#### POSTGRES_USER
 
-:   URL to represent the connection string to a database
+User to connect to the Postgres database used by Marsha.
 
-Type
+- Type: string
+- Required: No
+- Default: `"marsha_user"`
 
-:   String
+### Amazon Web Services-related settings
 
-Mandatory
+#### DJANGO_AWS_ACCESS_KEY_ID, DJANGO_AWS_SECRET_ACCESS_KEY
 
-:   No if `DJANGO_CONFIGURATION` is set to `Development`, yes otherwise
+A key ID + secret pair for an AWS IAM account with administrative access to the resources the relevant AWS deployment is using.
 
-Default
+- Type: string
+- Required: Yes
+- Default: None
 
-:   `sqlite:///path/to/project/db.sqlite3` if `DJANGO_CONFIGURATION` is
-    set to `Development`, None otherwise
+#### DJANGO_AWS_DEFAULT_REGION
 
-Choices
+The Amazon Web Services region where we deployed or want to deploy our serverless stack.
 
-:   See [schemas as presented by
-    dj-database-url](https://github.com/kennethreitz/dj-database-url#url-schema)
+- Type: string
+- Required: No
+- Default: `"eu-west-1"`
+- Choices: Any valid AWS region name.
+
+#### DJANGO_AWS_SOURCE_BUCKET_NAME
+
+The source AWS S3 bucket where files will be uploaded by end users. This should match the name of the bucket created by the relevant AWS deployment.
+
+- Type: string
+- Required: No
+- Default: Varies depending on the environment.
+  - `"development-marsha-source"` in development;
+  - `"test-marsha-source"` in test;
+  - `"staging-marsha-source"` in staging;
+  - `"preprod-marsha-source"` in preproduction;
+  - `"production-marsha-source"` in production.
+
+#### DJANGO_CLOUDFRONT_ACCESS_KEY_ID
+
+The ID of the AWS master account with which we want to sign urls (it is declared in the CloudFront distribution as a signing account).
+
+Note: Must be associated with a master account as IAM accounts cannot sign URLs.
+
+- Type: string
+- Required:
+  - Yes when `DJANGO_CLOUDFRONT_SIGNED_URLS_ACTIVE` is `True`;
+  - No otherwise.
+- Default: None;
+
+#### CLOUDFRONT_PRIVATE_KEY_PATH
+
+Path to a private key corresponding to the id in `DJANGO_CLOUDFRONT_ACCESS_KEY_ID`. Also used to sign Cloudfront URLs.
+
+- Type: string
+- Required:
+  - Yes when `DJANGO_CLOUDFRONT_SIGNED_URLS_ACTIVE` is `True` and the key is not located in the default path;
+  - No otherwise.
+- Default: `{REPO_ROOT}/.ssh/cloudfront_private_key`
```

Comment:
I think @lunika was moving it to the backend root in another PR.

Reply:
He actually changed the path to be this one for now.

Reply:
I closed the PR doing that so the path didn't changed.